index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html forms</title>
+    <link rel=“stylesheet”
+    href=“https://use.fontawesome.com/releases/v5.5.0/css/all.css”
+    integrity=“sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU”
+    crossorigin=“anonymous”>
```

Comment:
This is producing a web hint error. Please review font awesome instructions, as normally it is enough to add the Kit's Code to Your Project to work <script> tag.